lambda_example/sample_event.json

```diff
@@ -0,0 +1,8 @@
+{
+    "firstName": "Test",
+    "lastName": "Client",
+    "dateOfBirth": "5/1/1970",
+    "phoneNumber": "998877870",
+    "ssn": "223224549",
```

Comment:
@gustavodiel note that I changed ssn number to a string, because they can start out with 0, like 079-22-9876, and the zero would get lost yielding a SS that was a digit short.